code/modules/events/blowout.dm

```diff
@@ -55,8 +56,6 @@
 				LAGCHECK(LAG_LOW)
 				if (S.z == Z_LEVEL_STATION)
 					S.color = src.space_color
-				else
-					break
```

Comment:
I believe this may actually be fine due to how byond loops through the turfs, touching all z1 space before any other. This effectively stops it the moment it left the Z rather than having to loop through 4 other Z levels as well

Reply:
Related convo that happened a few days ago about the topic, mainly just trying to be safe with the code itself:
<img width="484" alt="breakpart1" src="https://user-images.githubusercontent.com/45720148/122299770-09e75700-cec4-11eb-9164-cdf2733203c1.png">
<img width="527" alt="breakpart2" src="https://user-images.githubusercontent.com/45720148/122299771-09e75700-cec4-11eb-99f0-87ac699f63ec.png">
<img width="480" alt="breakpart3" src="https://user-images.githubusercontent.com/45720148/122299772-0a7fed80-cec4-11eb-9747-22639616ea78.png">
<img width="494" alt="breakpart4" src="https://user-images.githubusercontent.com/45720148/122299774-0a7fed80-cec4-11eb-8ea5-3f2d9c12cc9c.png">
<img width="397" alt="breakpart5" src="https://user-images.githubusercontent.com/45720148/122299775-0a7fed80-cec4-11eb-95ca-f3db683db2ca.png">
<img width="387" alt="breakpart6" src="https://user-images.githubusercontent.com/45720148/122299776-0a7fed80-cec4-11eb-8199-30d7b04ff7c6.png">

Reply:
honestly its a terrible loop anyway. Looping through space in world and relying on what I assume to be undocumented behavior is dumb.

replace world with `block(locate(1, 1, Z_LEVEL_STATION), locate(world.maxx, world.maxy, Z_LEVEL_STATION))`
and ditch the else entirely